### PR TITLE
Debian package for /opt/openquake libs.

### PR DESCRIPTION
**Pull request by nastasi-oq**
Integrated with CI infrastructure, here an example: https://ci.openquake.org/job/zdevel_oq-libs/17
Creation of multiple repositories to use with hazardlib/engine packages CI jobs.
Same pattern to produce sources to upload on launchpad.
Binaries for different Ubuntu series must be done manually via web.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Source: python-oq-libs
+Section: python
+Priority: extra
+Maintainer: Matteo Nastasi (GEM Foundation) <nastasi@openquake.org>
+Build-Depends: debhelper (>= 7.0.50~), lsb-release, curl, python (>=2.7), virtualenv, python-virtualenv, python-wheel
```
Comment (daniviga) [inline debian/control]: Please remove virtualenv

Comment (daniviga) [inline debian/control]: Please remove `virtualenv`

File: debian/rules [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+#!/usr/bin/make -f
+# -*- makefile -*-
```
Comment (daniviga) [inline debian/rules]: Header is repeated

File: debian/control (hunk 1 above)
Comment (nastasi-oq) [inline debian/control]: Without it build fails.

File: debian/rules (hunk 2 above)
Comment (nastasi-oq) [inline debian/rules]: Done.

File: debian/control (hunk 1 above)
Comment (daniviga) [inline debian/control]: Launchpad install deps with `--no-install-recommends``, so that's the reason why.

**Pull request merged**